### PR TITLE
feat: split validate to teamsApp/validateManifest and teamsApp/validateAppPackage actions

### PR DESCRIPTION
**Pull request by nliu-ms**
For bug: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/16597713

Will provide 2 validation actions

### CODE REVIEW

**Comment by nliu-ms** [comment]
E2E TEST: https://github.com/OfficeDev/TeamsFx/actions/runs/4402430912/jobs/7709587817

**Review by HuihuiWu-Microsoft — APPROVED** [review]

File: templates/scenarios/js/m365-tab/teamsapp.yml.tpl [hunk 1]
```diff
@@ -84,7 +84,7 @@ deploy:
 
 # Triggered when 'teamsfx publish' is executed
 publish:
-  - uses: teamsApp/validate # This action is currently skipped, will be updated in the future version.
+  - uses: teamsApp/validateManifest # Validate using manifest schema
```
Comment (swatDong) [inline templates/scenarios/js/m365-tab/teamsapp.yml.tpl:87]: @kuojianlu - it seems no `teamsApp/validate` in m365tab's local yml? As designed or just missing one...

**Review by swatDong — APPROVED** [review]
Template change looks good to me.

**Review by jayzhang — APPROVED** [review]

File: templates/scenarios/js/m365-tab/teamsapp.yml.tpl (hunk 1 above)
Comment (kuojianlu) [reply templates/scenarios/js/m365-tab/teamsapp.yml.tpl:87]: Good catch. Seems missing. I will add it.

**Comment by kuojianlu** [comment]
The migration change will be in another PR?

**Pull request merged**